solara/components/echarts.vue

```diff
@@ -66,6 +83,11 @@ module.exports = {
         if (this.on_mouseout_enabled) this.on_mouseout(eventData);
       });
     },
+    handleResize() {
+  if (this.chart) {
+    this.chart.resize();
+  }
+},
```

Comment:
```suggestion
    handleResize() {
  if (this.chart) {
    this.chart.resize();
  }
},
```

I think `handleContainerResize` is a better name, what do you think? Could you please indent the code?